rclrs/src/lib.rs

```diff
@@ -205,6 +204,8 @@ mod test_helpers;
 
 mod rcl_bindings;
 
+include!(concat!(env!("OUT_DIR"), "/interfaces.rs"));
```

Comment:
This generates a _lot_ of warnings right now. Will need to look at how to hygienically do that (or move to another crate and suppress there)